### DISCUSSION TITLE
Unable to Complete Communicate using Markdown

### DISCUSSION THREAD

**Discussion by Yash-Shindey** [skills/.github#569] [Communicate using Markdown]
<img width="630" alt="Screenshot 2024-05-08 at 3 14 41 PM" src="https://github.com/skills/.github/assets/96872207/e8808ec3-168c-420b-be41-15b91de65c08">

Hey guys. I have done all the steps required and have published the page here https://exp-communicate-using-markdown-cohort-1.github.io/series-communicate-using-markdown-Yash-Shindey/  I am still unable to mark it as complete. Any help will be highly appreciated

**Comment by jules946** [comment]
Hi there,
I am also experienceing the same issue. 
Repo: https://github.com/Exp-Communicate-Using-Markdown-Cohort-1/series-communicate-using-markdown-jules946
![image](https://github.com/skills/.github/assets/114394404/8897e089-d59c-4314-ab75-ee8e68ad1f6f)
![image](https://github.com/skills/.github/assets/114394404/33598f52-abd8-45bb-8f5e-9cea3c0083b1)

**Comment by heiskr** [comment]
It looks like you're looking for GitHub Classroom which is a different product from GitHub Skills. I suggest you contact the GitHub support team instead.

**Comment by dxkf177** [comment]
[magisk_log_2024-05-21T18.01.37.log](https://github.com/skills/.github/files/15388713/magisk_log_2024-05-21T18.01.37.log)
[magisk_log_2024-05-21T18.43.14.log](https://github.com/skills/.github/files/15388716/magisk_log_2024-05-21T18.43.14.log)

**Comment by dxkf177** [comment]
[Uploading magisk_log_2024-05-22T10_38_59.log…]()
